packages/mui-material/src/OverridableComponent.d.ts

```diff
@@ -20,7 +20,7 @@ export interface OverridableComponent<M extends OverridableTypeMap> {
        */
       component: C;
     } & OverrideProps<M, C>,
-  ): JSX.Element;
+  ): JSX.Element | null;
   (props: DefaultComponentProps<M>): JSX.Element;
```

Comment:
I think this return type should also be `JSXElement | null`

Reply:
👍 Right, I missed it as well